### ISSUE TITLE
Cleanup of saperlipopette and other stuff in the infrastructure

### ISSUE THREAD

**Issue by alexAubin** [YunoHost/issues#1442]
So saperlipopette became a huge mess over time and we now have all those services which are outdated and should be removed : 
- The old redmine, we have some old documentation which imho is anyway irrelevant nowadays and my opinion is that we should just burn it
- chat.yunohost.org (the old xmpp chat that broke and we never found how to fix it and switchted to irc+matrix)
- doc.yunohost.org (some dokuwiki, I don't think we have any actual page...)
- the old mailman which we should just get rid of
- salade-de-patates and wekan which we never really used :/ we should install it because wekan takes some huge space
- the old weblate (nowadays it has its own container)
- im.yunohost.org, the xmpp chat logs ... do we really still need to maintain this considering that now stuff is logged on matrix or idk

Inside the ci-apps, there is also an old kanboard which I believed is unused...

On the bearnaise host, we have may stale nginx configs that should be removed, corresponding to the previously mentionned services.

Also one of the biggest issue of the infrastructure right now is related to the ci-apps container not cleaning swap files. As a consequence, we end up with something like 5+ Go taken by various files from reel2bits(?) and drupal (they are duplicated because of the multiinstance installs). Since it is the case for all the snapshot, the total amount to 15~20 Go of swap files.

**Comment by alexAubin** [comment]
ping @Bram and @maniackcrudelis in case you have some opinion or info about stuff that shouldn't be removed ... 

Otherwise, I'm going to go berserk some day and put fire on all of these.

**Comment by Psycojoker** [comment]
I'm pretty sure "im.yunohost.org" is heavily used by people that are only on XMPP :/

Appart from that I would be happy if we keep a backup of redmine somewhere but otherwise you can remove everything for me (salade-de-patates :cry: ...)

**Comment by maniackcrudelis** [comment]
Same as Bram for im.yunohost.org

The kanboard on ci-apps isn't used anymore I guess. And has no meaning since we gave up Official apps. (Even though it's not yet done).
The CI is supposed to remove its swap files now, are you sure that aren't old swap files ? If I can find some time I'll have a look to it.
But we crashed a CI because of the swap files of gitlab, since that, the CI is supposed to remove the swap file before any snapshot.
